packages/mux-video-react/src/index.tsx

```diff
@@ -105,7 +107,6 @@ MuxVideo.propTypes = {
   startTime: PropTypes.number,
   streamType: PropTypes.oneOf(Object.values(StreamTypes)),
   targetLiveWindow: PropTypes.number,
-  tokens: PropTypes.any,
```

Comment:
NOTE: `<mux-player>` and `<MuxPlayer>` have "tokens". `<mux-video>` now has `playbackToken` (as will `<mux-audio>`), in part for things like moving our more precise error logic into playback-core (potentially part of this PR effort). `toMuxVideoSrc()` util fn expects `tokens`, which is used for translating props to `src` URL. `<mux-video>` and `<mux-audio>` also relies on `playbackId` with extra query params (including `?token={PLAYBACK_TOKEN}`) to construct `src` URL... in other words, we should try to unwind some of this...

Reply:
Re-added as a feature for better consistency.